data/formats-data.js

```diff
@@ -2672,7 +2679,7 @@ exports.BattleFormatsData = {
 		tier: "BL",
 	},
 	surskit: {
-		randomBattleMoves: ["hydropump", "signalbeam", "hiddenpowerfire", "stickyweb", "gigadrain", "powersplit"],
+		randomBattleMoves: ["hydropump", "signalbeam", "scald", "stickyweb", "gigadrain", "icywind"],
```

Comment:
is icy wind slowing down the guy who doesnt have to switch into webs not enough?
if it isnt i'll do that

Reply:
icy wind just isnt used in LC at all, specially when all the speed tiers are so close together.  Surskit just sets webs and and either gets sacced or is saved to make sure webs stay up throughout the match

Reply:
I meant that it slows what kills you in case that mon came in before you set webs and wasn't slowed, but I guess this is less needed since you won't have full intentional webs teams with less speed